project/ScalafixBuild.scala

```diff
@@ -211,7 +207,7 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     // don't publish scala 3 artifacts for now
     publish / skip := (if ((publish / skip).value) true
                        else scalaBinaryVersion.value == "3"),
-    versionPolicyIntention := Compatibility.BinaryCompatible,
+    versionPolicyIntention := Compatibility.None,
```

Comment:
I want to merge this just before tagging 0.11.0, otherwise other breaking changes that we still want to minimize will go undetected